src/halmos/cheatcodes.py

```diff
@@ -405,12 +408,18 @@ def create_uint(ex, arg, **kwargs):
     if bits > 256:
         raise HalmosException(f"bitsize larger than 256: {bits}")
 
-    name = name_of(extract_string_argument(arg, 1))
+    try:
+        name = name_of(extract_string_argument(arg, 1))
+    except (IndexError, NotConcreteError):
+        name = "vmRandomUint"
     return ByteVec(uint256(create_generic(ex, bits, name, f"uint{bits}")))
 
 
 def create_uint256(ex, arg, **kwargs):
-    name = name_of(extract_string_argument(arg, 0))
+    try:
+        name = name_of(extract_string_argument(arg, 0))
+    except (IndexError, NotConcreteError):
+        name = "vmRandomUint256"
```

Comment:
would be better to avoid the try..catch pattern, maybe something with a default kwarg?